microsoft/testsuites/performance/common.py

```diff
@@ -80,3 +80,25 @@ def cleanup_process(environment: Environment, process_name: str) -> None:
     for node in environment.nodes.list():
         kill = node.tools[Kill]
         kill.by_name(process_name)
+
+
+def format_disk(
```

Comment:
It looks not like "format", it's like "delete".
`delete_partitions`

Reply:
This part of refactoring and were written as part of an earlier testcases. I will check with @LiliDeng to understand more about this function

Reply:
@LiliDeng Can you please confirm chi's suggestion that we should rename it to reset_partitions? It does seem like that we are deleting and re-creasing partitions which is essentially resetting it